_config/blinkr.yaml

```diff
@@ -151,6 +152,7 @@ ignores:
   - url: https://infinispan.ci.cloudbees.com/ #ISPN-4600
   - url: http://fisheye.jboss.org/browse/Guvnor #GUVNOR-2113
   - url: https://jboss.org/jbossas/osgi #JBAS-9573
+  - url: http://www.redhat.com/developers/rhel/RHEL_Developer_Getting_Started_Guide.pdf #RHD-1342
```

Comment:
This link can now be fixed. use: http://static.jboss.org/rhd/docs/rhel_developer_getting_started_guide.pdf